shared/js/background/classes/site.es6.js

```diff
@@ -131,6 +131,14 @@ class Site {
             return domain
         }
 
+        // for some reason chrome passes this back from webNavigation events
+        // for new tabs instead of chrome://newtab
+        //
+        // "local-ntp" -> "local new tab page"
+        if (url.match(/^chrome-search:\/\/local-ntp/)) {
+            return 'new tab'
+        }
```

Comment:
I'm not sure why Chrome returns this weird URL in the webNavigation event. Seems we can handle it pretty easily though.

Reply:
Thanks for including this! It had been bugging me for a while.